mllib/src/main/scala/org/apache/spark/ml/regression/GBTRegressor.scala

```diff
@@ -153,20 +158,21 @@ object GBTRegressor {
  * @param _trees  Decision trees in the ensemble.
  * @param _treeWeights  Weights for the decision trees in the ensemble.
  */
+@Since("1.4.0")
 @Experimental
-final class GBTRegressionModel(
-    override val uid: String,
-    private val _trees: Array[DecisionTreeRegressionModel],
-    private val _treeWeights: Array[Double])
+final class GBTRegressionModel @Since("1.4.0") (
+                                                 @Since("1.4.0") override val uid: String,
+                                                 private val _trees: Array[DecisionTreeRegressionModel],
+                                                 private val _treeWeights: Array[Double])
```

Comment:
Fix the indentations for the parameters. There is no problem about the `@Since` tag.